src/generate_acs_major_group.ado

```diff
@@ -9,15 +9,15 @@ __generate_acs_major_group__ {hline 2} Generate categorical variable for major i
 Description
 -----------
 
-__generate_acs_major_group__ generates a categorical variable for major industry and major occupation groups in American Community Survey [public use microdata](https://www.census.gov/programs-surveys/acs/technical-documentation/pums.html) by recoding _occp_ and _indp_. 
+__generate_acs_major_group__ generates a categorical variable for major industry and major occupation groups in American Community Survey [public use microdata](https://www.census.gov/programs-surveys/acs/technical-documentation/pums.html) by recoding 'occp' and 'indp'. 
```

Comment:
Link not displaying correctly.

Reply:
Could you reply with screenshot?

Reply:
<img width="565" alt="link_issue" src="https://user-images.githubusercontent.com/55054385/90527264-585e8480-e13f-11ea-8e88-1812299718b7.PNG">

Reply:
I think this is a typo that I fixed but that you are not seeing because you are viewing via `help` (which I believe would open the installed version of the package help files) instead of opening the `.sthlp` file in the repo directory (the most up to date version). Could you try the latter and see if the link is still incorrect?

Reply:
Oh weird yes, that resolves it.